src/richie/apps/search/forms.py

```diff
@@ -317,7 +329,7 @@ def build_es_query(self):
         if full_text:
             query = {
                 "query": {
-                    "match": {"title.fr": {"query": full_text, "analyzer": "french"}}
+                    "match": {"title.fr": {"query": full_text, "analyzer": "simple"}}
```

Comment:
this would still not be accurate for organizations and categories. Actually we should just remove the analyzer spec from this query so that it just uses the same analyzer as specified in the mapping for each object...